pkgs/development/tools/build-managers/buildbot9/plugins.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv,
+  fetchurl,
+  buildPythonApplication,
+  python
+}:
+
+{
+  www = buildPythonApplication rec {
+    name = "buildbot-www-0.9.0rc1";
+    namePrefix = "";
+    format = "wheel";
```

Comment:
We cannot handle circular dependencies, and in my opinion having such circular dependency would be a bug. Anyway, using a wheel seems like a good solution.